### PR TITLE
Add a bio for @FloodFloodington

### PR DESCRIPTION
**Pull request by FloodFloodington**
Adds a bio for @FloodFloodington in response to issue #6. 

I'd love to have @jci-trawicki review this. I won't use emoji here - I don't want to insult your intelligence.

Wait, I better soften things here :hankey:. Gross.

### CODE REVIEW

File: docs/FloodFloodington.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# FloodFloodington
+
+**Name:** Dennis Flood
```
Comment (FloodFloodington) [inline docs/FloodFloodington.md:3]: Consider adding an initial here for middle name

File: docs/FloodFloodington.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+**Occupation:** User Experience Designer
+
+**Hometown:** Athens, Ontario
```
Comment (jeremyhannon) [inline docs/FloodFloodington.md]: Perhaps add a country specifier for Ontario.

Comment (FloodFloodington) [inline docs/FloodFloodington.md]: Great idea.

**Comment by FloodFloodington** [comment]
:+1:

**Comment by jeremyhannon** [comment]
:ship: it.

**Pull request merged**